backend/src/config/archives.json

```diff
@@ -0,0 +1,6 @@
+[
+    {
+        "url": "wss://kusama-rpc.polkadot.io",
+        "path": "/Users/apple/Documents/kusama-archive-2021-oct-23"
```

Comment:
This is not what I meant. I meant that these files shouldn't be in the repo at all, there should be instructions and examples in the readme on how to run it because this path only works on your machine and nowhere else.

Not blocking for now, but required for this code to be runnable by anyone else.

Reply:
I will update this in the next PR